code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -59,7 +59,8 @@
 	if(can_collar)
 		dat += "<tr><td>&nbsp;</td></tr>"
 		dat += "<tr><td><B>Collar:</B></td><td><A href='?src=[UID()];item=[slot_collar]'>[(collar && !(collar.flags&ABSTRACT)) ? collar : "<font color=grey>Empty</font>"]</A></td></tr>"
-
+	if(facehugger)
+		dat += "<tr><td><B>Facehugger:</B></td><td><A href='?src=[UID()];["remove_hugger"]=remove_hugger'>[facehugger]</A></td></tr>"

```

Comment:
`["remove_hugger"]=remove_hugger` I'm not even sure what you're trying to do here, it should just be `;remove_hugger=1`.